<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditFamilyView</class>
 <widget class="QDialog" name="EditFamilyView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>567</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>449</width>
    <height>567</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>449</width>
    <height>567</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>تعديل بيانات عائله</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>427</width>
     <height>548</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_49">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_50">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_51">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_52">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_53">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_22">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_54">
                <item>
                 <widget class="QLabel" name="label_27">
                  <property name="text">
                   <string>اسم الزوج</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_fatherName">
                  <property name="frame">
                   <bool>false</bool>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_55">
                <item>
                 <widget class="QLabel" name="label_28">
                  <property name="text">
                   <string>اسم الزوجه</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_motherName">
                  <property name="frame">
                   <bool>false</bool>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_56">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_23">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_57">
              <item>
               <widget class="QLabel" name="label_29">
                <property name="text">
                 <string>شقه (*)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_flatNo_3">
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="maxLength">
                 <number>3</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_58">
              <item>
               <widget class="QLabel" name="label_30">
                <property name="text">
                 <string>مدخل/مربع (*)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_entranceNo_3">
                <property name="text">
                 <string/>
                </property>
                <property name="maxLength">
                 <number>3</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_59">
              <item>
               <widget class="QLabel" name="label_31">
                <property name="text">
                 <string>عماره (*)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_buildingNo_3">
                <property name="text">
                 <string/>
                </property>
                <property name="maxLength">
                 <number>3</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_24">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_60">
              <item>
               <widget class="QLabel" name="label_32">
                <property name="text">
                 <string>قطعه (*)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_blockNo_3">
                <property name="text">
                 <string/>
                </property>
                <property name="maxLength">
                 <number>3</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_61">
              <item>
               <widget class="QLabel" name="label_33">
                <property name="text">
                 <string>المجاوره (*)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_neighbourhoodNo_3">
                <property name="text">
                 <string/>
                </property>
                <property name="maxLength">
                 <number>3</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_62">
              <item>
               <widget class="QLabel" name="label_34">
                <property name="text">
                 <string>نوع السكن</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_residenceType_3">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_25">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_63">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_64">
            <item>
             <widget class="QLabel" name="label_35">
              <property name="text">
               <string>اسم الابن / الابنه</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_toAddChildName_3">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="locale">
               <locale language="Arabic" country="Egypt"/>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAtBottom</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_26">
            <item>
             <widget class="QPushButton" name="pushButton_addChild_3">
              <property name="text">
               <string>اضافه ابناء</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_65">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_66">
            <item>
             <widget class="QLabel" name="label_36">
              <property name="text">
               <string>اسماء الابناء المضافه</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_toDeleteChildName_3">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="locale">
               <locale language="Arabic" country="Egypt"/>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_deleteChild_3">
            <property name="text">
             <string>حذف</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_27">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_67">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_28">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_68">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <item>
               <widget class="QLabel" name="label_37">
                <property name="text">
                 <string>اسم المستضاف</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_toAddResidentName_3">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="editable">
                 <bool>true</bool>
                </property>
                <property name="insertPolicy">
                 <enum>QComboBox::InsertAtBottom</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_69">
              <item>
               <widget class="QLabel" name="label_38">
                <property name="text">
                 <string>نوع القرابه</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_residentRelation_3">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_29">
            <item>
             <widget class="QPushButton" name="pushButton_addResident_3">
              <property name="text">
               <string>اضافه مستضاف</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_70">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_71">
            <item>
             <widget class="QLabel" name="label_39">
              <property name="text">
               <string>اسماء المستضافين المضافه</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_toDeleteResidentName_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_deleteResident_3">
            <property name="text">
             <string>حذف</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_note_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16770000</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="locale">
         <locale language="Arabic" country="Egypt"/>
        </property>
        <property name="autoFormatting">
         <set>QTextEdit::AutoAll</set>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string>ملاحظات</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_30">
      <item>
       <widget class="QPushButton" name="pushButton_applyFamilyEdit">
        <property name="text">
         <string>حفظ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_mainWinReturn_4">
        <property name="text">
         <string>رجوع</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
